package-deploy.yaml

```diff
@@ -24,6 +24,24 @@ agents:
       - AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
     workDir: $HOME/projects/go-algorand
 
+  - name: deb
+    dockerFilePath: docker/build/aptly.Dockerfile
+    image: algorand/aptly
+    version: scripts/configure_dev-deps.sh
+    buildArgs:
+      - GOLANG_VERSION=`./scripts/get_golang_version.sh`
+    env:
+      - AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID
+      - AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
+      - CHANNEL=$CHANNEL
+      - SNAPSHOT=$SNAPSHOT
+      - STAGING=$STAGING
+      - VERSION=$VERSION
+    volumes:
+      - $XDG_RUNTIME_DIR/gnupg/S.gpg-agent:/root/.gnupg/S.gpg-agent
```

Comment:
Not to fix here, but this is probably one of those things we'll have to look at with Mac clients.

Reply:
Agree, it's on my list :)